gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/jdbc/authentication/LDAPAuthenticationSchemeImpl.java

```diff
@@ -631,6 +629,12 @@ private String getDNFromUID(String uid)
 			throw new NameNotFoundException();
 			
 		SearchResult result = (SearchResult)results.next();
+
+		if (GemFireXDUtils.TraceAuthentication) {
+			SanityManager.DEBUG_PRINT(AuthenticationServiceBase
```

Comment:
Please correct the indentation. ( 2 space thing )

Reply:
@kneeraj  will correct it. As for your query , I will get back tomorrow.